### DISCUSSION TITLE
Option for listening_timout in seconds?

### DISCUSSION THREAD

**Discussion by Bataghast** [talonhub/community#1584] [Q&A]
I'm often talking to a handful of people at once while working I would like to address talon explicitly when giving a command instead of manually going in and out of sleep mode.

I'm trying to get a setup where I just say the word "Talon" (waking talon up) then my command, followed by talon immediately going back into sleep mode. user.listening_timeout_minutes is close but it's a minimum of one minute, I need something closer to three to five seconds.

This is what I have so far:

```
mode: sleep
mode: command
-
settings():
    # Enable the Talon mode indicator
    user.mode_indicator_show = true
	
    # Turn off Talon if no commands after several minutes
    user.listening_timeout_minutes = 1

^(talon [listen]) :
    speech.toggle()

key(f9):
    speech.toggle()
```

**Comment by auscompgeek** [comment]
A hack, but you'd probably prefer something like https://github.com/tararoys/presentation_mode

**Reply by Bataghast** [reply]
Oh thank you, I'll check it out!
